stdlib/messaging/kafka/src/main/java/org/ballerinalang/messaging/kafka/impl/KafkaListenerImpl.java

```diff
@@ -83,6 +95,38 @@ public void onRecordsReceived(ConsumerRecords records,
     @Override
     public void onError(Throwable throwable) {
         logger.error("Kafka Ballerina server connector retrieved exception: " + throwable.getMessage(), throwable);
+        String url = ((Properties) listener.getNativeData(NATIVE_CONSUMER_CONFIG)).getProperty(BOOTSTRAP_SERVERS);
```

Comment:
Why this line has to repeat all the time? Can't we resolve URL at the init and then re-use it?